### DISCUSSION TITLE
Galaxy brain achiev

### DISCUSSION THREAD

**Discussion by hussainkelarno** [Schweinepriester/github-profile-achievements#573] [Q&A] [answered]
I tried to answer a couple of questions but the achievement didn't show up, how did I get this achievement?

**Comment by hussainabdillah** [comment, marked as answer]
> same here with me, I think you should email it to GitHub cs to complain 

edit : I found a solution, if you try to mark yourself your answer before, try to unmark it. because it is affect to the giving of achievement
